test/integration/test/philips_light_test.js

```diff
@@ -0,0 +1,99 @@
+// This agent refers to PORT where program is runninng.
+const spawn = require('child_process').spawn;
+const chakram = require('chakram'), expect = chakram.expect;
+const Config = require('config-js');
+
+var philipshue_server = require('../lib/philipsHue_server.js');
+var nupnp_server = require('../lib/nupnp_PhilipsHue.js');
+var upnp_server = require('../lib/upnp_PhilipsHue.js');
+var config = new Config('./lib/config/foxbox.js');
+
+describe("Initiate the connection with foxbox as Philips Hue hub",function(){
+  var credential = config.get('credential');
+  var key = credential['username']+":"+credential['password'];
+  var encoded_cred = new Buffer(key).toString('base64');
+  var header = config.get('header');
+  var lightinfo; // to store the light status received from foxbox
+  var FOXBOX_STARTUP_WAIT_TIME_IN_MS = 5000;
+  var foxbox_process;
+  before(function(done){
+    this.timeout(500000);
+
+    // Start the nupnp server
+    const hue_location = "localhost:" + config.get('philips_hue.port');
+    nupnp_server.start(config.get('nupnp_server.id'),hue_location,config.get('nupnp_server.port'));
+    
+    // Start the Philips Hue simulator, and set the #2 lightbulb status to false
+    philipshue_server.setup(config.get('philips_hue.port'));
+    philipshue_server.setlight(2,false);
+
+    // kick off the foxbox within 10 seconds
+    remove_sqlite = spawn('rm',['users_db.sqlite']);
```

Comment:
Nit: Use [fs.unlink](http://stackoverflow.com/questions/5315138/node-js-remove-file), which is cross-platform